models/issues/issue_list.go

```diff
@@ -310,6 +312,7 @@ func (issues IssueList) loadAssignees(ctx context.Context) error {
 
 	for _, issue := range issues {
 		issue.Assignees = assignees[issue.ID]
+		issue.isAssigneeLoaded = true
```

Comment:
If I understand correctly, in issue.LoadAssignees we will set issue.assignee, but in issuelist.LoadAssignees we will not do this. is it correct?
https://github.com/go-gitea/gitea/pull/30490/files#diff-d3a8b9a325e353adebf461f60e7a34be0be36306a8f83f56d3c8cf372f24c346R46-R49

Reply:
d21752894906e7cdd92753cf9cb5c453412e5ab8